work_queue/src/work_queue_json.c

```diff
@@ -392,3 +392,33 @@ char *work_queue_json_remove(struct work_queue *q, int id)
 	return task;
 
 }
+
```

Comment:
Looks fine, except please indent consistently with the rest of the code.
Use `../../cctools.indent` if necessary.